metabase/metabase
# adds minimal embed focused homepage

[[Epic] Meet embedders at the door - minimal edition](https://github.com/metabase/metabase/issues/38233)

### Description

This adds a the minimal homepage from [this design](https://www.figma.com/file/UFay4YjWyUMyl23T1cXzeC/Meet-first-time-embedders-at-the-door-with-a-dashboard?type=design&node-id=1449-15221&mode=design&t=C56e5BXR4bEjeJAk-4).
For the tests, I only modified a current e2e to expect to not find the homepage, and then I made a new flow that selects "Interested in embedding" and expects to find it.

*I will add more tests in a later PR*

### How to verify

1. Do the setup, select that you're interested in embedding or "both"
2. You should see the embedding homepage
3. You should be able to dismiss it and it should go away and never come back again

### Demo

<img width="1374" alt="image" src="https://github.com/metabase/metabase/assets/1914270/bb1063c1-818d-424c-b027-a6e910097daa">

### Checklist
